src/cryptonote_basic/miner.cpp

```diff
@@ -401,7 +410,13 @@ namespace cryptonote
     
     for(size_t i = 0; i != m_threads_total; i++)
     {
-      m_threads.push_back(boost::thread(m_attrs, boost::bind(&miner::worker_thread, this)));
+      if (threads_idle) {
+        m_threads.push_back(create_background_thread(m_attrs, boost::bind(
+            &miner::worker_thread, this)));
+      } else {
+        m_threads.push_back(boost::thread(m_attrs, boost::bind(
+            &miner::worker_thread, this)));
+      }
```

Comment:
Minor gripe, but formatting should follow the rest of the file (here, braces on newlines).